po/es.po

```diff
@@ -356,58 +355,58 @@ msgid "(AI) detect from image edges..."
 msgstr ""
 
 #: ../build/lib/darktable/plugins/introspection_channelmixerrgb.c:499
-#: ../src/iop/channelmixerrgb.c:1669
+#: ../src/iop/channelmixerrgb.c:1804
 #, fuzzy
 msgid "as shot in camera"
 msgstr "Buscar cámara"
```

Comment:
"as shot in camera" is translated to:
```suggestion
msgstr "Buscar cámara"
```
Should be "Como fue tomado en cámara" ?

Reply:
Thank you, David. I appreciate the suggestion you made, however, it would be necessary to add writing style and take into account other details.

La razón de que la veas así es porque está en modo borrador y no es una traducción final. Si usas la versión de desarrollo, notarás que ni siquiera aparece así. Tal vez encuentres algunas otras líneas como esta y la razón la explico en #6818.